packages/admin-next/dashboard/src/routes/locations/location-detail/components/location-general-section/location-general-section.tsx

```diff
@@ -69,49 +74,49 @@ export const LocationGeneralSection = ({
         locationId={location.id}
         locationName={location.name}
         type={FulfillmentSetType.Pickup}
-        fulfillmentSet={location.fulfillment_sets.find(
+        fulfillmentSet={location.fulfillment_sets?.find(
```

Comment:
Currently has the exact same functionality as the Shipping fulfillment set, e.g. you can add return and outbound shipping options, but is this the intended behaviour? Seems to be based on Figma design, but in that case what does it have to do with Pickup?

Reply:
Added a comment on this in Figma. This is intended behavior – but perhaps we should consider renaming the concept before the official release. However, I don't think it's super high priority right now.